src/pages/UserProfile.tsx

```diff
@@ -138,16 +178,77 @@ const CustomSpan = styled.span`
   font-size: 1.3rem;
   text-align: center;
 `
- 
+
 const CustomButton = styled(ButtonPrimary)`
   margin: 1rem;
 `
 const Actions = styled.div`
   text-align: center;
 `
+const IcBtnCamera = styled(IconButton)`
```

Comment:
```suggestion
const CameraIconButton = styled(IconButton)`
```
có mấy chữ cũng viết tắt làm gì e? Tên component chứ phải class đâu